docs/topics/analyzer-rbac-snippet.adoc

```diff
@@ -0,0 +1,9 @@
+// snippet
+
+[WARNING]
+.No multi-user access restrictions on resources
+====
+There are no multi-user access restrictions on resources.
+
+An analyzer task created by a user, can be canceled by any other user.
```

Comment:
I suggest to remove the specific example of analyzer task access,
as we have the no multi-user access restriction on other resources as well.
Just leave the ".No multi-user access restrictions on resources" warning
and place it in the head of the MTA doc.